extensions/notebook/src/test/common/port.test.ts

```diff
@@ -53,7 +53,7 @@ describe('Ports', () => {
 				options.maxRetriesPerStartPort = 1;
 				options.totalRetryLoops = 50;
 				ports.strictFindFreePort(options).then(freePort => {
-					assert.ok(freePort >= 7100 && freePort !== initialPort);
+					assert(freePort !== initialPort, `Expected freePort !== initialPort, Actual: ${freePort}`);
```

Comment:
do you also want to print initial port?

Reply:
I believe in this scenario it's the same as freePort.